vita3k/gui/src/game_install_dialog.cpp

```diff
@@ -33,13 +33,13 @@
 
 namespace gui {
 
-nfdchar_t *game_path = nullptr;
-static bool draw_file_dialog = true;
-static bool game_install_confirm = false;
-
 void draw_game_install_dialog(GuiState &gui, HostState &host) {
+    static nfdchar_t *game_path = nullptr;
     nfdresult_t result = NFD_CANCEL;
 
+    static bool draw_file_dialog = true;
+    static bool game_install_confirm = false;
```

Comment:
NO GOD PLEASE NO

Reply:
For ? I have set it here for not set this outside like petmac wanted  and is perfect like this 🤔